stable/insights-agent/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 4.3.2
+* added prometheus flag to skip non zero validation
```

Comment:
Since we are adding a new parameter, I think we should bump the **minor version**